docroot/profiles/custom/cgov_site/config/install/adobe_dtm.settings.yml

```diff
@@ -0,0 +1,5 @@
+enabled: true
+property_embed_code_id: f1bfa9f7170c81b1a9a9ecdcc6c5215ee0b03c84
```

Comment:
Hrrmm... so this config will apply to all sites, but I am sure we will want multiple. So how can we handle this in a multi site situations?

Reply:
Changed the embed_code_hash to a development property. The embed_code_id is the same for all NCI sites.